esp8266/uart.h

```diff
@@ -96,5 +96,6 @@ int uart_rx_char(void);
 void uart_tx_one_char(uint8 uart, uint8 TxChar);
 void uart_flush(uint8 uart);
 void uart_os_config(int uart);
+void uart_setup(uint8 uart_no);
```

Comment:
Minor, but as you can see above, this argument name is called "uart" for the other functions, and consistency should rather be preserved.